lib/rules/applet-alt.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "applet-alt",
+  "selector": "applet",
+  "matches": "",
+  "tags": [
+    "cat.text-alternatives",
+    "wcag2a",
+    "wcag111"
+  ],
+  "metadata": {
+    "description": "Checks if a text alternative is provided for an applet",
```

Comment:
Description should be in the active tense, like "Applet must have a text alternative"